nohup: ignoring input
setcovering-small
firstsol
asymmetric
seed: 101
cpu
Traceback (most recent call last):
  File "compute_evaluation_results.py", line 77, in <module>
    reinforce_localbranch.primal_integral(test_instance_size=instance_size, total_time_limit=total_time_limit, node_time_limit=node_time_limit)
  File "/home/liudefen/projects/ML4LocalBranch_extend/mllocalbranch_fromfiles.py", line 7546, in primal_integral
    with gzip.open(filename, 'rb') as f:
  File "/home/liudefen/opt/python3.8.0/lib/python3.8/gzip.py", line 58, in open
    binary_file = GzipFile(filename, gz_mode, compresslevel)
  File "/home/liudefen/opt/python3.8.0/lib/python3.8/gzip.py", line 173, in __init__
    fileobj = self.myfileobj = builtins.open(filename, mode or 'rb')
FileNotFoundError: [Errno 2] No such file or directory: './result/generated_instances/setcovering/-small/asymmetric/rootsol/rl/reinforce/test/evaluation-reinforce4lb-from-rootsol-t_node10s-t_total60s-small/rlactive/seed101/lb-test-setcovering-160_transformed.pkl'
